app/graphql/types/model.rb

```diff
@@ -0,0 +1,11 @@
+module Types
+  class Model < GraphQL::Schema::Object
+    description 'A Model'
+
+    field :id, Integer, resolve: ->(obj, _, _) { obj.id }, null: false
+    field :name, String, resolve: ->(obj, _, _) { obj.name }, null: false
+    field :info, String, resolve: ->(obj, _, _) { obj.info }, null: true
+    field :vendorClass, String, resolve: ->(obj, _, _) { obj.vendor_class }, null: true
+    field :hardwareModel, String, resolve: ->(obj, _, _) { obj.hardware_model }, null: true
```

Comment:
You can remove the `:resolve` and change to `:hardware_model`, it will work. Same for other fields.